archive/dto.ksy

```diff
@@ -0,0 +1,57 @@
+meta:
+  id: dto
+  title: DTB/DTBO
+  license: CC0-1.0
+  file-extension: img
+  endian: be
+
+doc-ref:
+  - https://source.android.com/devices/architecture/dto/partitions
+  - https://android.googlesource.com/platform/system/libufdt/+/refs/heads/master/utils/
```

Comment:
Can you craft a permanent link to a specific version? The tag `master` changes over time, so after 1-2 years it won't be clear what you were referring to (if one won't rely on internet archives to dig up what `master` is referring to at _this_ time). Moreover, they change the directory structure of the repository and the `utils/` folder will no longer exist, for example. So please, either link to a commit hash or some version tag so that we're future proof.